lib/syslog-ng.h

```diff
@@ -47,6 +47,8 @@
 typedef struct _LogPipe LogPipe;
 typedef struct _LogMessage LogMessage;
 typedef struct _GlobalConfig GlobalConfig;
+typedef struct _CfgLexer CfgLexer;
+typedef struct _PluginContext PluginContext;
```

Comment:
Third definition

Reply:
I have removed the extra ones, thanks for noticing.